scene/animation/animation_mixer.cpp

```diff
@@ -1975,6 +1982,7 @@ void AnimationMixer::_build_backup_track_cache() {
 				if (t_obj) {
 					t->value = Animation::cast_to_blendwise(t_obj->get_indexed(t->subpath));
 				}
+				t->use_continuous = true;
```

Comment:
Why is use_continuous necessarily true in `_build_backup_track_cache`? There seems to be no check here: What if the track was discrete?

Reply:
In the case of discrete, values may not be processed depending on the key position, and the scene values may be rewritten when saving.